integration_tests/test_dict_02.py

```diff
@@ -27,7 +35,6 @@ def test_dict():
     for i in range(1000, 1000 + size, 7):
         assert abs(rollnumber2cpi[i] + f64(i)/100.0 + 5.0) <= 1e-12
 
-
     assert abs(rollnumber2cpi[0] - 1.1) <= 1e-12
 
 test_dict()
```

Comment:
I would add new tests rather than modifying the existing ones, as I think we want to be testing both.

Reply:
Done with the changes. Reverted the changes in original test cases. Made new test cases and added them to `CmakeLists`. @certik